### ISSUE TITLE
Did you try weight_decay, dropout and softmax for classify?

### ISSUE THREAD

**Issue by k3832127** [brantbzhang/tensorflow-mtcnn#3]
I think weight decay and dropout using in train step will improve cnn performance. 
Did you try this?
Waiting for you answer.

**Comment by brantbzhang** [comment]
I think data is enough and network is very simple, so not easy to overfitting.You can add test accuracy to verify it.
